lib/floe/workflow/runner/kubernetes.rb

```diff
@@ -35,6 +35,8 @@ def initialize(options = {})
 
           @namespace = options.fetch("namespace", "default")
 
+          @workflow_service_account = options["workflow_service_account"]
```

Comment:
Naming is hard - couldn't figure out a good name here, but if you have other ideas, please mention.

Reply:
Agreed on task_service_account, since this is specifically for the tasks that are launched.